packages/google_identity_services_web/lib/src/js_interop/google_accounts_id.dart

```diff
@@ -230,12 +230,40 @@ abstract class IdConfiguration {
     /// the callback is invoked.
     Function? intermediate_iframe_close_callback,
 
-    /// determines if the upgraded One Tap UX should be enabled on browsers
+    /// Determines if the upgraded One Tap UX should be enabled on browsers
     /// that support Intelligent Tracking Prevention (ITP). The default value
     /// is false.
     ///
     /// See: https://developers.google.com/identity/gsi/web/guides/features#upgraded_ux_on_itp_browsers
     bool? itp_support,
+
+    /// If your application knows in advance which user should be signed-in, it
+    /// can provide a login hint to Google.
+    ///
+    /// When successful, account selection is skipped. Accepted values are:
+    ///   * an email address or
+    ///   * an ID token sub field value.
+    ///
+    /// For more information, see:
+    ///   * https://developers.google.com/identity/protocols/oauth2/openid-connect#authenticationuriparameters
+    String? login_hint,
```

Comment:
The ship has clearly already sailed on this, but out of curiosity why are we using snake case instead of camel case for all of these, given that it's Dart code?

Reply:
> why are we using snake case instead of camel case for all of these, given that it's Dart code?

Originally, my idea was that it'd be easier for users to find documentation for the parameters if they were called exactly the same as in the underlying JS SDK (see also some of the [exotic names of the WebGL API in package:web](https://github.com/dart-lang/web/blob/main/lib/src/dom/ext_blend_minmax.dart)).

If we prefer `camelCase` in the Dart side of things, it can definitely be done:

```dart
@JS('login_hint')
String? loginHint,
```

And it wouldn't be a hard migration, after all I suspect it's only [us using this package directly](https://pub.dev/packages?q=dependency%3Agoogle_identity_services_web+show%3Aunlisted).

Let me know if you want this camelCase'd, I can create a task and go for it later :)

Reply:
I don't know that it's worth doing a breaking change for, it just struck me as odd so I was curious.